08-regression2.Rmd

```diff
@@ -453,41 +463,94 @@ get our test error.
 lm_mult_test_results
 ```
 
-We get that the RMSPE for the multivariate linear regression model 
-of `r format(lm_mult_test_results %>% filter(.metric == 'rmse') %>% pull(.estimate), scientific = FALSE)`. This prediction error
+We obtain an RMSPE for the multivariate linear regression model 
```

Comment:
For beta_0 above, again, I think we need to specify it's the value of y where x and z are 0.

Reply:
done